app/views/CreditAndRefunds.scala.html

```diff
@@ -202,41 +195,32 @@ <h2 class="govuk-heading-s">@{
                     }
 
                     @{
-                        if(maybeBalancesDetails.firstPendingAmountRequested > maybeBalancesDetails.secondPendingAmountRequested){
-                            maybeBalancesDetails.firstPendingAmountRequested map { amount =>
-                                if(messages.lang.language.toLowerCase().equals("cy")){
-                                        <li>
-                                        <p class="govuk-heading-s">
+                        if(balance.get.firstPendingAmountRequested > balance.get.secondPendingAmountRequested){
+                            balance.get.firstPendingAmountRequested map { amount =>
+                                <li>
+                                    <p class="govuk-heading-s">
                                         <span class="govuk-body">{messages("credit-and-refund.refundProgress-prt-1")}</span>
                                         {amount.toCurrencyString}
                                         <span class="govuk-body">{messages("credit-and-refund.refundProgress-prt-2")}</span>
-                                        </p>
-                                        </li>
-                                }
-                                else{
-                                        <li>
-                                        <p class="govuk-heading-s">{amount.toCurrencyString}
-                                        <span class="govuk-body">{messages("credit-and-refund.refundProgress")}</span>
-                                        </p>
-                                        </li>
-                                }
+                                    </p>
+                                </li>
                             }
                         }
                         else {
-                            maybeBalancesDetails.secondPendingAmountRequested map { amount =>
+                            balance.get.secondPendingAmountRequested map { amount =>
```

Comment:
pls use headOption instead, as this is not safe to call get directly / use MissingFieldException is required